Src/Sarif.PatternMatcher/ValidatingVisitor.cs

```diff
@@ -38,7 +38,7 @@ public override Result VisitResult(Result node)
                 // {0:scanTarget}' contains {1:validationPrefix}{2:encoding}{3:secretKind}{4:validationSuffix}{5:validatorMessage}
                 string message = null;
 
-                FailureLevel level = node.Level;
+                FailureLevel level = default;
```

Comment:
>level [](start = 29, length = 5)

This is right! and a good change. Seeing this I think I understand the bug, was NoMatch getting the default level??